apps/aecore/src/aec_chain_server.erl

```diff
@@ -107,6 +107,26 @@ handle_call(difficulty, _From, State) ->
 handle_call({common_ancestor, Hash1, Hash2}, _From, State) ->
     {reply,
      aec_chain_state:find_common_ancestor(Hash1, Hash2,State), State};
+handle_call(get_genesis_header, _From, State) ->
+    GenesisHash = aec_chain_state:get_genesis_hash(State),
```

Comment:
I see the genesis hash is kept in the state so to optimize the retrieval of genesis. As a "get-by-height-0" would otherwise traverse the chain from top all the way down. Ok.